applications/NXstress.nxdl.xml

```diff
@@ -0,0 +1,1115 @@
+                    </doc>
+                </item>
+                <item value="sin2psi">
+                    <doc>
+                        A description of the :math:`\mathrm{\sin}^{2}\psi` method can be found in the literature. Two examples are:
+                        `Fitzpatrick, M. E. et. al.; Determination of residual stresses by X-ray diffraction; National Physical Laboratory, Teddington, 2005`
+                        and
+                        `EN 15305:2009 standard: Non-destructive testing - Test Method for Residual Stress analysis by X-ray Diffraction`.
+                    </doc>
+                </item>
+            </enumeration>
+        </field>         
+        <field name="measurement_direction" type="NX_CHAR" minOccurs="0" maxOccurs="1">
+            <doc>
+                Describes the specific measurement direction covered by the data in this file.
+            </doc>
+            <enumeration>
+                <item value="radial"/>
+                <item value="longitudinal"/>
+                <item value="normal"/>
+                <item value="tangential"/>
+                <item value="multiple"/>
+            </enumeration>
+        </field>
+        <group name="experiment_responsible" type="NXuser" minOccurs="0" maxOccurs="1">
+            <doc>Information about the person who performed the experiment.</doc>
+            <field name="name" type="NX_CHAR" minOccurs="0" maxOccurs="1">
+            </field>
+            <field name="role" type="NX_CHAR" minOccurs="0" maxOccurs="1">
+                <doc>Role of user responsible for this entry. Suggested roes are, for example, ``local contact``, ``beamline_scientist``, ``post_doc``,…</doc>
+            </field>
+        </group>
+        <group name="instrument" type="NXinstrument" minOccurs="1" maxOccurs="1">
+            <field name="name" type="NX_CHAR" minOccurs="1" maxOccurs="1">
+                <doc>Name of the diffractometer, instrument, or beamline used for the experiment. This could be, for example, *Strain Analyser for Large and Small scale engineering Applications*.</doc>
+                <attribute name="short_name" optional="true">
+                    <doc>Short name for the instrument, perhaps the acronym, which would be for the the example above ``SALSA``.</doc>
+                </attribute>
+            </field>
+            <group name="CALIBRATION" nameType="any" type="NXnote" minOccurs="0" maxOccurs="1">
+                <doc>This group contains information about the geometry and/or efficiency measurement(s).</doc>
+                <field name="calibration_type" type="NX_CHAR" minOccurs="0" maxOccurs="1">
+                    <doc>Describe the type of calibration.</doc>
+                </field>
+                <field name="file_name" type="NX_CHAR" minOccurs="0" maxOccurs="1">
+                    <doc>File name(s) and/or path(s) (withing file(s)) containing data from the last calibration(s). This can be a list.</doc>

```

Comment:
```suggestion
                    <doc>File name(s) and/or path(s) (within file(s)) containing data from the last calibration(s). This can be a list.</doc>

```